character/tw.js

```diff
@@ -274,6 +275,161 @@ game.import('character',function(lib,game,ui,get,ai,_status){
 			},
 		},
 		skill:{
+			//张昭
+			twlijian:{
+				audio:2,
+				sunben:true,
```

Comment:
建议将技能标签风格统一，推荐使用类似于`elationSkill`的名称
```suggestion
				elationSkill:true,
```